zcash_test_vectors/zip_0233.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python3
+import sys; assert sys.version_info[0] >= 3, "Python 3 required."
+
+from hashlib import blake2b
+import struct
+
+from .transaction import TransactionV6
+from .output import render_args, render_tv, Some
+from .rand import Rand
+from .zip_0143 import (
+    SIGHASH_ALL,
+    SIGHASH_ANYONECANPAY,
+    SIGHASH_NONE,
+    SIGHASH_SINGLE,
+)
+
+from .zip_0244 import *
+
+def main():
+    args = render_args()
+
+    from random import Random
+    rng = Random(0xB7D6_0F44)
+    def randbytes(l):
+        ret = []
+        while len(ret) < l:
+            ret.append(rng.randrange(0, 256))
+        return bytes(ret)
+    rand = Rand(randbytes)
```

Comment:
```suggestion
    rand = Rand(randbytes(rng))
```

Function exist and in scope for this.